tests/TestCase/ExceptionsTest.php

```diff
@@ -172,7 +172,7 @@ public function exceptionProvider()
             ['Cake\Datasource\Exception\MissingModelException', 500],
             ['Cake\Datasource\Exception\PageOutOfBoundsException', 404],
             ['Cake\Datasource\Exception\RecordNotFoundException', 404],
-            ['Cake\Mailer\Exception\MissingActionException', 404],
+            ['Cake\Mailer\Exception\MissingActionException', 500],
```

Comment:
Is anyone familiar with this exception. It seemed like a bug that calling the wrong mailer method would throw a 404 error. If this is intentional for some reason, let me know.

Reply:
I agree, that this might have been done based on "missing action" being controller related on normal URLs.
But for mailer it sure sounds like it should be a 500.